const blockController = {}

blockController.addBlock = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}

blockController.blocks = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.addBlock = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.addReason = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.Reason = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.Reasons = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.updateReason = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}
blockController.deleteReason = async (req, res) => {
  try {
  } catch (error) {
    res.status(500).json({succes: false, message: 'Internal Server Error'})
  }
}

module.exports = blockController
